<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>About</class>
 <widget class="QWidget" name="About">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About &quot;Global&quot;</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/g_icons/global1.png</normaloff>:/g_icons/global1.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 85, 225);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons/resources.qrc">:/global.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>4</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Bookman Old Style</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff;
margin-bottom:0;</string>
       </property>
       <property name="text">
        <string>Global</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons/resources.qrc">:/global.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_8">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bookman Old Style</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(254, 255, 246);</string>
     </property>
     <property name="text">
      <string>calculating charge state distributions</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_14">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(222, 226, 197); background-color: rgb(0, 90, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>&lt;br&gt;W. E. Meyerhof&lt;hr&gt;C. Scheidenberger, Th. St√∂hlker, W. E. Meyerhof,&lt;br&gt;H. Geissel, P. H. Mokler, B. Blank&lt;br&gt;Nucl. Instr. and Meth. B 142 (1998) 441&lt;br&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_Version">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(240, 255, 200);</string>
     </property>
     <property name="text">
      <string>Version 1.5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_Date">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(240, 255, 200);</string>
     </property>
     <property name="text">
      <string>28-April-2015</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="ClickableLabel" name="label_mail">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>send mail to:</string>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 5px;
color: rgb(210, 210, 210);</string>
     </property>
     <property name="text">
      <string>&lt;p&gt;GLOBAL has been ported to MS Windows and C++&lt;br&gt; by O.B. Tarasov  within the framework of the program LISE&lt;sup&gt;++&lt;/sup&gt;.&lt;/p&gt;&lt;p&gt;This program has been made into a cross-platform &lt;br&gt; application in the Qt framework by M.P. Kuchera.&lt;/p&gt;&lt;p&gt;The GUI-version is currently maintained by O.B.Tarasov&lt;/p&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 90, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="horizontalSpacing">
       <number>9</number>
      </property>
      <property name="verticalSpacing">
       <number>12</number>
      </property>
      <item row="0" column="1" colspan="2">
       <widget class="ClickableLabel" name="label_Charge">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Link to Charge&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(222, 226, 197);</string>
        </property>
        <property name="text">
         <string>http://web-docs.gsi.de/~weick/charge_states</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(170, 170, 255);</string>
        </property>
        <property name="text">
         <string>LISE&lt;sup&gt;++&lt;/sup&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(170, 170, 255);</string>
        </property>
        <property name="text">
         <string>Global</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ClickableLabel" name="label_LISE">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;link to LISE++&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(222, 226, 197);</string>
        </property>
        <property name="text">
         <string>http://lise.nscl.msu.edu</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="icons/resources.qrc">:/emblem_little_plus.png</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>w_Stuff/w_Label_clickable.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons/resources.qrc"/>
 </resources>
 <connections/>
</ui>
